src/containers/CreatePipelineRun/CreatePipelineRun.test.js

```diff
@@ -130,6 +130,14 @@ describe('CreatePipelineRun', () => {
     expect(queryByDisplayValue(/bar/i)).toBeFalsy();
   });
 
+  it('renders yaml mode', () => {
+    const { getByRole } = renderWithRouter(<CreatePipelineRun />, {
+      path: '/create',
+      route: '/create?mode=yaml'
+    });
+    expect(getByRole(/cm-content/)).toBeTruthy();
```

Comment:
This test is failing, guess you meant to use something other than `getByRole` here